Fix accessibility issues

nickbristow opened the pull request:
Fixes 1,658 accessibility issues, 

I've done some testing, but should be verified by someone more familiar with the app

Commit:
added lang element,
fixed contrast errors

Commit:
added unique_id method helpers to handlebars,
added get_conntent_response unique ids and changed how its selected

Commit:
added unique_id method helpers to handlebars,
added get_conntent_response unique ids and changed how its selected

Commit:
added aria-label to param text fields

Commit:
added aria-label to param text fields

Commit:
fixed duplicated id bug

LindsayYoung commented:
This fixes a ton of issues. Thanks again Nick for doing this!

I noticed that it breaks links to anchor tags, I will see if I can get in a quick fix. If not, I can approve this PR and file an issue to fix the anchor links later.

LindsayYoung commented on src/main/template/operation.handlebars:
```diff
@@ -4,15 +4,17 @@
         <h3>
           <span class='http_method'>
-          <a href='#!/{{encodedParentId}}/{{nickname}}' class="toggleOperation">+</a>
+          <a href='#{{parentId}}_{{nickname}}' data-href='#!/{{encodedParentId}}/{{nickname}}' class="toggleOperation">+</a>
```
There are 2 hrefs here

noahmanger replied:
Ah. Well one of those is `data-href`, but i don't see where that's being used. What's this for @nickbristow ?

Commit:
Wanted to make the links buttons, as they behave like buttons
  Changed links to buttons
  Gave them a link-button class to style them like links
  added a click handler to main to update the url using the data-href

nickbristow commented:
@noahmanger @LindsayYoung ok take a look now, I think this is clearer???

LindsayYoung commented:
Yes, I like this. The section headings don't have links, but the endpoints do, and I think that is more important. 

Making buttons into buttons is brilliant, as far as I am concerned. 

Thanks again!
